src/core/co_types.h

```diff
@@ -25,6 +25,12 @@
 #include <stdbool.h>
 #include <stdint.h>
 
+// For microcontrollers that do not have uint8_t/int8_t
+#ifdef __TMS320C28XX__
+typedef uint16_t uint8_t;
+typedef int16_t int8_t;
```

Comment:
Instead of redefining the standard types I recommend using uint_least8_t and int_least8_t in this code. These types are generally defined to be the smallest available native size capable of accommodating an 8b value